initialize_internal_tide.jl

```diff
@@ -17,26 +17,29 @@ function initialize_internal_tide(
     Nx,
     Ny,
     Nz;
+    Δtᵒ,
+    tᶠ,
+    θ,
+    U₀,
+    N,
     closure = SmagorinskyLilly(),
-    θ = 3.6e-3,
-    U₀ = 0.025,
+    output_mode,
+    output_writer,
     topo_file = "topo.mat"
+    
 )
 
 
 function log_gpu_memory_usage()
 # Capture the output of CUDA.memory_status()
-    output = IOBuffer()
-    CUDA.memory_status(output)
+    # output = IOBuffer()
+    mem_info_str = CUDA.memory_status()
```

Comment:
Why is this `IOBuffer()` commented out? We should not have any commented-out lines of code. If we don't need the lines of code, just delete them. If you want to have the option to include the line of code back in later, then use a keyword argument and a conditional `if` statement.

Reply:
I'll delete it, for some reason, when writing `mem_info_str = CUDA.memory_status(output)`, I get errors that I haven't gotten before.

Reply:
We upgraded some packages since the last time you ran GPU simulations, didn't we?

Reply:
yeah, we upgraded Oceananigans